src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  font-family: Arial, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+  padding: 20px;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Consider adding a CSS rule for `.stars--0` to handle cases where no stars should be active. This will ensure consistency with the HTML structure and prevent any unexpected styling issues.